pkg/reconciler/delivery/delivery.go

```diff
@@ -99,53 +111,120 @@ func (c *Reconciler) updateRoute(ctx context.Context, cfg *v1.Configuration) err
 	if err != nil {
 		return err
 	}
+
 	logger.Info("Applying updated Route object")
 	_, err = c.client.ServingV1().Routes(cfg.Namespace).Update(route)
 
+	// when we have latestRevision = true, we know that we don't need to queue future events
```

Comment:
Why don't we check err here?

Reply:
You're definitely right! This function was implemented differently previously and I didn't check err previously because it would be directly returned below. However, now that some other logic takes place before `return err` I think you're right in that we should check err here.